guides/common/modules/proc_securing-the-dhcp-api.adoc

```diff
@@ -0,0 +1,26 @@
+[id="Securing_the_dhcpd_API_{context}"]
+= Securing the dhcpd API
+
+By default, the dhcpd API listens to any host.
+You can add a `omapi_key` to provide basic security.
+
+.Procedure
+. On Enterprise Linux 8, generate a key:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# dnf install -y bind-utils
+# dnssec-keygen -r /dev/urandom -a HMAC-MD5 -b 512 -n HOST omapi_key
+# cat Komapi_key.+*.private | grep ^Key|cut -d ' ' -f2-
+----
+. Use `{foreman-installer}` to secure the dhcpd API:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} \
+--foreman-proxy-dhcp-key-name "_My_Name_" \
+--foreman-proxy-dhcp-key-secret "_My_Secret_"
+----
+. In the {ProjectWebUI}, navigate to *Infrastructure > Smart Proxies*.
+. Select your {SmartProxy}.
+. On the *Overview* tab, click *Refresh* in the *Actions* menu.
```

Comment:
The installer will refresh if needed. In this particular case it's not even needed since it only affects dhcpd.
```suggestion
```

Reply:
Removed.